src/sections/Meshery/Meshery-integrations/IntegrationsGrid.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import HoneycombGrid from "./HoneycombGrid.style";
+
+function IntegrationsGrid({ obj }) {
+  return (
+    <HoneycombGrid>
+      <div className="container">
+        {obj.map((item) => {
+          const status = item.status === "InProgress" ? true : false;
+          if (status) {
+            return (
```

Comment:
`return` statement not needed

Reply:
I tried removing the return statement, but it didn't show the Integration Grid section.